pipelinewise/fastsync/commons/target_snowflake.py

```diff
@@ -60,12 +60,18 @@ def open_connection(self):
                                            autocommit=True,
                                            session_parameters={
                                                # Quoted identifiers should be case sensitive
-                                               'QUOTED_IDENTIFIERS_IGNORE_CASE': 'FALSE'
+                                               'QUOTED_IDENTIFIERS_IGNORE_CASE': 'FALSE',
+                                               'QUERY_TAG': json.dumps({
+                                                   'ppw_component': 'fastsync',
+                                                   'tap_id': self.connection_config.get('tap_id'),
+                                                   'table': query_tag_props.get('schema'),
+                                                   'schema': query_tag_props.get('table')
```

Comment:
Looks like there is a typo here: `'schema': query_tag_props.get('table')` and above too.
`schema` value is being set with value of `table` key and vice versa.

Reply:
fixed by 26febe82084667ac7e7db47299147e0dfa284257